src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -41,7 +42,9 @@
  *
  * @author depristo
  */
-public final class CommonInfo {
+public final class CommonInfo implements Serializable {
```

Comment:
Yes, I've added docs to the `VariantContextBuilder` attribute setting methods warning that attribute values must be serializable or else serialization will fail. Unfortunately we're probably not at liberty to change the actual type from `Object` to `Serializable`, or to throw if a value if not an instance of `Serializable`, as these would be breaking changes for client code.